tests/components/smlight/conftest.py

```diff
@@ -18,6 +19,20 @@
 MOCK_PASSWORD = "test-pass"
 
 
+@pytest.fixture
+def mock_async_step_reauth() -> Generator[MagicMock, None, None]:
+    """Mock the async_step_reauth method."""
+    with patch(
+        "homeassistant.components.smlight.config_flow.SmlightConfigFlow.async_step_reauth",
+        return_value={
+            "type": data_entry_flow.FlowResultType.FORM,
+            "flow_id": "mock_flow",
+            "step_id": "reauth_confirm",
+        },
+    ) as mock:
+        yield mock
```

Comment:
What's this for?

Reply:
this was to test that reauth was triggered, but maybe simpler/clearer to do it inline in the test.